mcs/class/System/Mono.Net.Security/SystemCertificateValidator.cs

```diff
@@ -45,7 +45,8 @@ static SystemCertificateValidator ()
 #elif MONODROID
 			is_macosx = false;
 #else
-			is_macosx = System.IO.File.Exists (OSX509Certificates.SecurityLibrary);
+			// make it work on wine too.
+			is_macosx = Environment.OSVersion.Platform == PlatformID.MacOSX;
```

Comment:
We used many version of mono and the `is_macosx` field moved from class to class, then the 3 `CertificateValidatorTypes`